x/proof/keeper/keeper.go

```diff
@@ -63,10 +63,19 @@ func NewKeeper(
 
 	ringClient, err := rings.NewRingClient(ringClientDeps)
 	if err != nil {
-		return Keeper{}, err
+		panic(err)
```

Comment:
#PUC explaining rationale. Is this because of an expectation that `NewKeeper()` should only be called on startup?

Reply:
OK, I think I understand your use case for making `NewKeeper`s return an error. But then shouldn't we remove the `sdk.AccAddressFromBech32(authority)` panic from the other `Keeper`'s initializers?